assets/sass/modules/_comments.scss

```diff
@@ -0,0 +1,23 @@
+.comments__area {
+  margin-top: 50px;
+  padding: 50px;
+  background: $gray;
+  & h3 {
+    @include fontSize(24px);
+
+  }
+  & input , & textarea {
+    padding: 10px;
+    margin: 0;
+    outline: 0;
+    border: 2px solid white;
```

Comment:
Try to follow 1 rule here you are using color names instead of using hex values, please use variables to references colours in the theme.

Thanks.